### PR TITLE
check_number_of_missing_files_per_rse ported to Prometheus

### PR DESCRIPTION
**Pull request by FernandoGarzon**

### CODE REVIEW

File: cms/check_number_of_missing_files_per_rse [hunk 1]
```diff
@@ -21,35 +21,66 @@
...
+        for rse, num in results:
+            #print(type(rse), rse, type(num), num)
+            prom_labels = {'rse': rse}
+            prom_labels.update(extra_prom_labels)
+            missing_files_gauge.labels(**prom_labels).set(num)
+            monitor.record_gauge(
+                name='judge.number_of_missing_files.%s' % rse,
+                value=num)
+            #print(rse, num)
+        if PROM_SERVERS:
```
Comment (ericvaandering) [inline cms/check_number_of_missing_files_per_rse:78]: No need to push things twice. At the end of the probe is fine.

File: cms/check_number_of_missing_files_per_rse [hunk 2]
```diff
@@ -21,35 +21,66 @@
...
+                 .filter(models.Request.request_type == RequestType.TRANSFER)
+                 .filter(models.ReplicationRule.state != RuleState.OK).group_by(models.RSE.rse))
         results = query.all()
         print('Number of missing files per RSE')
-        for rse, num in result:
-            monitor.record_gauge(stat='judge.number_of_missing_files.%s' % rse, value=num)
-            print(rse, num)
+        for rse, num in results:
+            #print(type(rse), rse, type(num), num)
+            prom_labels = {'rse': rse}
```
Comment (ericvaandering) [inline cms/check_number_of_missing_files_per_rse:71]: If you look here: https://github.com/dmwm/rucio-flux/blob/main/infrastructure/statsd/statsd-prometheus-mapping.yaml you'll see that the prometheus label is dst_rse not rse. We should keep it the same.

Same for the second gauge, of course.

File: cms/check_number_of_missing_files_per_rse [hunk 3]
```diff
@@ -21,35 +21,66 @@
...
-    status = OK
-    session = get_session()
+    registry = CollectorRegistry()
+    labelnames = ['rse']
+    labelnames.extend(extra_prom_labels.keys())
+    missing_files_gauge = Gauge('rucio_judge_number_of_missing_files','Number of missing files in a RSE',
+                             labelnames=labelnames, registry=registry)
+    replicating_files_gauge = Gauge('rucio_judge_number_of_replicating_files', 'Number of files being replicated to a RSE',
+                                labelnames=labelnames, registry=registry)
+    #status = OK
```
Comment (ericvaandering) [inline cms/check_number_of_missing_files_per_rse:57]: I don't think there is any reason to change the exit codes from what they originally were

**Review by ericvaandering — CHANGES_REQUESTED** [review]

**Comment by ericvaandering** [comment]
@FernandoGarzon it looks like you addressed all the comments. Can you confirm?

**Comment by FernandoGarzon** [comment]
I did, @ericvaandering

**Pull request merged**